solr/CHANGES.txt

```diff
@@ -212,6 +212,8 @@ Bug Fixes
 
 * SOLR-14122: SimUtils converts v2 to v1 request params incorrectly. (Li Cao, ab)
 
+* SOLR-6613: TextField.analyzeMultiTerm does not throw an exception when Analyzer returns no term. (Bruno Roustant)
```

Comment:
"no term" -> "no terms"

Reply:
Hum right, this strange rule of negation and plural. There is the same notion in French and I don't like it neither. To me it means "not a single term" or "zero term" so why should we put plural.
But ok I'll change that.